DuckDuckGoTests/FeedbackModelTests.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  FeedbackModelTests.swift
+//  DuckDuckGo
+//
+//  Copyright © 2017 DuckDuckGo. All rights reserved.
```

Comment:
Should we be putting 2018 in our copy right now?  I probably do this as well tbh :)